drivers/i2c/Kconfig

```diff
@@ -210,7 +211,7 @@ config I2C_0_DEFAULT_CFG
 
 config I2C_0_IRQ_PRI
 	int "Port 0 interrupt priority"
-	depends on I2C_0 && !HAS_DTS_I2C
+	depends on I2C_0 && !HAS_DTS_I2C && !I2C_ESP32
```

Comment:
Not understsanding this one.  Can't you just turn off HAS_DTS_I2C for the esp32 board if the board configuration doesn't include the I2C controller in the devicetree configuration?  Isn't that the point of HAS_DTS_I2C in the first place?

Reply:
The I2C device drivers are being converted to use DTS: #755. Since this is a new driver it would be best if it already used DTS. Adding DTS support is not much work. All these Kconfig dependencies will become much simpler.